pages/dkp/konvoy/1.8/release-notes/1.8.4/index.md

```diff
@@ -15,6 +15,8 @@ enterprise: false
 
 <p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. For new customers, contact your sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download Konvoy.</p>
 
+<p class="message--warning"><strong>WARNING: </strong>This version includes an update to D2iQ Dispatch that removes Argo CD. Do NOT update the version of Dispatch if you are currently relying on Argo CD that is part of Dispatch. D2iQ Dispatch has been deprecated in favor of Flux. See the <a href="https://d2iq.com/blog/goodbye-dispatch-hello-fluxcd">D2iQ blog post</a> for more information.</p>
```

Comment:
```suggestion
<p class="message--warning"><strong>WARNING: </strong>Konvoy 1.8.4 includes an update to D2iQ Dispatch that removes Argo CD. Do NOT apply the Konvoy 1.8.4 patch if you rely on Argo CD that is part of Dispatch. D2iQ Dispatch has been deprecated in favor of Flux. See the <a href="https://d2iq.com/blog/goodbye-dispatch-hello-fluxcd">D2iQ blog post</a> for more information.</p>
```

Reply:
Its possible to upgrade the version of Konvoy, Kubernetes-Base-Addons, Kommander without bumping Dispatch.
Im not sure if we want to block users from completely upgrading.

Reply:
We discussed this in the bug meeting, and that's what we said but let's check with PM. @sonali-mishra-22 or @chessahoekstra

Reply:
Dispatch is provided by the following addon:

    - configRepository: https://github.com/mesosphere/kubeaddons-dispatch
      configVersion: stable-1.20-1.4.6

The stable-1.20-1.4.6 version of kubeaddons-dispatch is what was shipped with Konvoy 1.8.4, and this is what removes the ArgoCD tool.   As Dmitri says, it is possible to upgrade  Konvoy, KBA and Kommander WITHOUT upgrading the dispatch addon.   So this is what we should tell people.   Here's my suggestion:

```suggestion
<p class="message--warning"><strong>WARNING: </strong>This version includes an update to the D2iQ Dispatch addon (stable-1.20-1.4.6) that removes Argo CD. Do NOT update to this version of Dispatch if you are currently relying on Argo CD that is part of Dispatch, instead, ensure that the  D2iQ Dispatch version in your config.yaml is stable-1.20-1.4.5.   D2iQ Dispatch has been deprecated in favor of Flux. See the <a href="https://d2iq.com/blog/goodbye-dispatch-hello-fluxcd">D2iQ blog post</a> for more information.</p>
```

Reply:
If approved by support, this LGTM.

Reply:
Looks good to me